CHANGELOG.md

```diff
@@ -35,6 +36,12 @@ Changelog for NeoFS Node
 - `morph.switch_interval` IR and SN config value is not used anymore.
 - `morph.rpc_endpoint` SN config value and `morph.endpoint.client` IR config value has been deprecated and will be 
   removed with the next minor release. Use `morph.endpoints` for both instead (NOTE: it does not have priorities now).
+- If you're using binary keys with neofs-cli (`-w`), convert them to proper
```

Comment:
Please, could you duplicate this manual to README or any place where it'd be easy to find?

Reply:
I think it's only relevant for this upgrade and careful admins read this documentation when they perform upgrade. And frankly I don't expect to see anyone still using these binaries keys, they're not even easy to create now (everyone just goes `neo-go wallet create` route).